### ISSUE TITLE
keep_prio_summary text is irritating, correction suggested

### ISSUE THREAD

**Issue by sphenotux** [mpcjanssen/simpletask-android#1149]
Standard todotxt behaviour is to **remove** priority (A), (B), (C) from done tasks, [see documentation](https://github.com/todotxt/todo.txt#rule-2-the-date-of-completion-appears-directly-after-the-x-separated-by-a-space):

> Many Todo.txt clients discard priority on task completion.

(Also linked and [discussed here](https://github.com/todotxt/todo.txt-cli/issues/323#issuecomment-731606054).)

Current behaviour in 10.9.3 depends on settings:
When a task with a prio is marked as "done",

- prio is **dropped** - if _Settings - Others_ (?) - _Keep priority in completed tasks_ is **unchecked**
- prio **remains** - if _Settings - Others_ (?) - _Keep priority in completed tasks_ is **checked**

IMHO, **header** of _keep_prio_ is correct. But _keep_prio_**_summary**_ of the _keep_prio_ option says:

> Todo.txt removes priority on completion

Suggestion: Change _removes_ to _keeps_. This fits the checkbox state.

Long story short: make _keep_prio_summary_ 

> Todo.txt keeps priority on completion

PR will follow for English and German language version.

**Comment by sphenotux** [comment]
Thank you for merging, @mpcjanssen! IMHO this issue may be closed, **but** translations to other languages than EN+DE open.
